framework/midi/internal/midiportdatasender.cpp

```diff
@@ -53,8 +53,6 @@ bool MidiPortDataSender::sendEvents(tick_t fromTick, tick_t toTick)
         const Event& event = pos->second;
 
         if (SKIP_EVENTS.find(event.type) == SKIP_EVENTS.end()) {
-            // noop
```

Comment:
Shouldn't that `==` turn into a `!=` then?

Reply:
It was a bug. Now it works as it should: send event only if it is not in the skip list.
```
if (SKIP_EVENTS.find(event.type) == SKIP_EVENTS.end()) {
    midiOutPort()->sendEvent(event);
}
```